econml/ortho_forest.py

```diff
@@ -461,6 +475,25 @@ class ContinuousTreatmentOrthoForest(BaseOrthoForest):
         `fit` and `predict` methods. If parameter is set to ``None``, it defaults to the
         value of `model_Y` parameter.
 
+    global_residualization : bool, optional (default=False)
+        Whether to perform a prior residualization of Y and T using the model_Y_final and model_T_final
+        estimators, or whether to perform locally weighted residualization at each target point.
+        Global residualization is computationally less intensive, but could lose some statistical
+        power, especially when W is not None.
+
+    global_res_cv : int, cross-validation generator or an iterable, optional (default=2)
+        The specification of the cv splitter to be used for cross-fitting, when constructing
+        the global residuals of Y and T.
+
+    discrete_treatment : bool, optional (default=False)
```

Comment:
I know. I was worried about this too, but thought it's fine to do it this way. Alternatively we can move all the discretization of treatment stuff in the causal forest fit function and not allow for discrete treatment here. Though this limits functionality: what if I want to use local residualization and the dml moments (not the DR moments that are used in DiscreteTreatmentORF)? This is done here with discrete_treamtent=True. So I thought we could add it.

I was alternatively compensating renaming to:
DMLOrthoForest
DROrthoForest

instead of ContinousTreatmentOrthoForest and DiscreteTreatmentOrthoForest and deprecating the old names in the next release. 

What do you think of this??

Reply:
I thought we were going to have just a single OrthoForest with the discrete_treatment flag, and then we could add deprecated aliases for both Continuous and Discrete that inherit from it and set the flag appropriately; is there a reason to keep both classes separate?

Reply:
I'm changing them to DML and DR orthoforest and decprecating the old ones